roundware/api2/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
```

Comment:
cool...since this PR doesn't pass Travis.ci, there will certainly need to be some updates, so you can add these changes then, as you say.